modules/ballerina-tools/conf/composer-config.yml

```diff
@@ -0,0 +1,9 @@
+#port: 9093
```

Comment:
Shall we remove the file or content if not used ?

Reply:
@hevayo Used this as sample config, shall we add this to a readme then?

Reply:
@hevayo added a new line to top explaining the structure